CMakeLists.txt

```diff
@@ -11,8 +11,10 @@ if(UNIX)
 endif()
 
 if(WIN32)
-	set(CMAKE_LIBRARY_OUTPUT_DIRECTORY_DEBUG "${CMAKE_SOURCE_DIR}/test/plugins")
-	set(CMAKE_LIBRARY_OUTPUT_DIRECTORY_RELEASE "${CMAKE_SOURCE_DIR}/test/plugins")
+	if(MVC)
```

Comment:
Why is this additional check required?

Reply:
Sorry, I left that in by mistake when I was testing different compilers on Windows, making sure they all compiled and outputted to the correct folders. Was testing to see how the each one would act when the left to it's defaults and when forced to use a certain output folder. (wanted to know if it was required to use the output on every compiler) Turns out only MVC needs it, since all other compilers don't create `Release` and `Debug` folders and will follow `set(CMAKE_BINARY_DIR ${CMAKE_SOURCE_DIR}/test/plugins)` as the output folder. 

So yeah that line only supposed to affect MVC due to the folders it creates.